src/endpoints/mod_versions.rs

```diff
@@ -283,6 +299,29 @@ pub async fn update_version(
             return Err(ApiError::DbError);
         }
     };
+    let fetched_mod = Mod::get_one(path.id.as_str(), false, &mut transaction).await?;
+    if let Some(valid_mod) = fetched_mod {
+        let version = valid_mod.versions.last().unwrap();
+        let is_update = valid_mod.versions.len() > 1;
+        let owner = &valid_mod.developers[0];
+
+        println!("{}", is_update);
+
+        send_webhook(
```

Comment:
this webhook should be sent after the mod version is actually updated, and only if the old status is pending and the new status is accepted